# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-core/Cargo.toml
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 third_party/rust/wgpu-core/Cargo.toml
--- a/third_party/rust/wgpu-core/Cargo.toml	Fri Oct 03 18:01:30 2025 +0000
+++ b/third_party/rust/wgpu-core/Cargo.toml	Fri Oct 03 18:08:29 2025 +0000
@@ -129,7 +129,7 @@
 version = "0.2.10"
 
 [dependencies.hashbrown]
-version = "0.15"
+version = "0.16"
 features = [
     "default-hasher",
     "inline-more",
@@ -137,7 +137,7 @@
 default-features = false
 
 [dependencies.indexmap]
-version = "2.7"
+version = "2.8"
 default-features = false
 
 [dependencies.log]
@@ -165,7 +165,7 @@
 default-features = false
 
 [dependencies.ron]
-version = "0.10"
+version = "0.11"
 optional = true
 
 [dependencies.rustc-hash]
@@ -182,7 +182,7 @@
 default-features = false
 
 [dependencies.smallvec]
-version = "1.13.1"
+version = "1.14"
 
 [dependencies.thiserror]
 version = "2.0.12"